bin/build.sh

```diff
@@ -249,7 +258,7 @@ then
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
     echo "deb http://download.mono-project.com/repo/ubuntu trusty main" | sudo tee /etc/apt/sources.list.d/mono-official.list
     sudo apt-get update
-    sudo apt-get install -y ${DEPENDENCIES}
+    sudo apt-get install -y --force-yes ${DEPENDENCIES}
```

Comment:
Wait, aren't `-y` and `--force-yes` synonyms?

Reply:
[Actually, no](https://linux.die.net/man/8/apt-get). One of the Objective-C packages does something deemed undesirable by apt, so simply using -y does not work here. I'm not sure exactly which package does it - this is something I'd like to look into when I fix the symlink in the future.

Reply:
Oh, man, here is what man pages say about this:
```
Force yes. This is a dangerous option that will cause apt-get to continue without prompting if it is doing something potentially harmful. It should not be used except in very special situations. Using --force-yes can potentially destroy your system!
```
I would like to avoid having `force-yes` here. A workaround would be that you do not extend `DEPENDENCIES` but rather install them separately in the `INSTALL_OBJECTIVEC` if condition. Does that make sense?

Reply:
Yes, this is the original strategy I was using before the review. I will revert it when I get time.